src/main/java/com/opentable/server/EmbeddedJettyBase.java

```diff
@@ -305,6 +312,17 @@ public void gracefulShutdown(ContextClosedEvent evt) {
         LOG.debug("Received application context closed event {}. Shutting down...", evt);
         LOG.info("Early shutdown of Jetty connectors on {}", container);
         if (container != null) {
+            if(shouldSleepBeforeShutdown) {
+                long sleepDurationMillisBeforeShutdown = sleepDurationBeforeShutdown.toMillis();
+                LOG.info("Application config requesting sleep for {} ms before Jetty shutdown", sleepDurationMillisBeforeShutdown);
+                try {
+                    LOG.info("Sleeping for {} ms before jetty shutdown", sleepDurationMillisBeforeShutdown);
+                    Thread.sleep(sleepDurationMillisBeforeShutdown);
+                } catch (InterruptedException e) {
+                    LOG.error("Failed to sleep before shutdown {}", e);
+                    Thread.currentThread().interrupt();
+                }
+            }
```

Comment:
Add a line after  - LOG.info("Stopping Jetty...") 

That way it's possible to observe the transitions better.